src/AWS.Deploy.Orchestration/CdkProjectHandler.cs

```diff
@@ -14,7 +14,7 @@ namespace AWS.Deploy.Orchestration
     public interface ICdkProjectHandler
     {
         public Task CreateCdkDeployment(OrchestratorSession session, CloudApplication cloudApplication, Recommendation recommendation);
-        public Task<string> CreateCdkProjectForDeployment(Recommendation recommendation, OrchestratorSession session, string? saveDirectoryPath = null);
+        public string CreateCdkProjectForDeployment(Recommendation recommendation, OrchestratorSession session, string? saveDirectoryPath = null);
```

Comment:
remove "public" since you are updating the interface

Reply:
Fixed